pc/io.go

```diff
@@ -111,9 +112,43 @@ L_HEADER:
 		return nil, errors.New("count field size is wrong")
 	}
 
-	switch fmt {
+	switch ppFmt {
 	case Ascii:
-		panic("not implemented yet")
+		pp.Data = make([]byte, pp.Points*pp.Stride())
+		offset := 0
+		for {
+			line, _, err := rb.ReadLine()
+			if err != nil && err != io.EOF {
+				return nil, err
+			}
+			if err == io.EOF {
+				break
+			}
+			pointData := strings.Fields(string(line))
+			for i, f := range pp.Type {
+				if f == "F" {
+					v, err := strconv.ParseFloat(pointData[i], 32)
+					if err != nil {
+						return nil, err
+					}
+					b := math.Float32bits(float32(v))
+					binary.LittleEndian.PutUint32(
+						pp.Data[offset:offset+4], b,
+					)
+				}
+
+				if f == "U" {
+					v, _ := strconv.ParseUint(pointData[i], 10, 32)
+					if err != nil {
+						return nil, err
+					}
+					binary.LittleEndian.PutUint32(
+						pp.Data[offset:offset+4], uint32(v),
+					)
+				}
+				offset += pp.Size[i]
```

Comment:
The result goes wrong if `COUNT` is not 1.

Reply:
Is it currently possible to read the values of a field for which `COUNT` is > 1? Considering a file like the following, can we use a `pp.Float32Iterator("vfh")` to read the `308` values of the field `vfh`?
```
# .PCD v0.7 - Point Cloud Data file format
VERSION 0.7
FIELDS vfh
SIZE 4
TYPE F
COUNT 308
WIDTH 1
HEIGHT 1
VIEWPOINT 0 0 0 1 0 0 0
POINTS 1
DATA ascii
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 99.097015 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
```
I might be wrong but I think actually supporting `COUNT` > 1 requires quite some changes. We could add a check when reading the header to reject such files for now.

Reply:
`FIELDS=xyz TYPE=F COUNT=3` can be used as vectors by https://github.com/seqsense/pcgol/pull/6

At least, offset of other fields shouldn't be affected by non-1 COUNT.